.gitignore

```diff
@@ -6,3 +6,4 @@
 venv/
 .pytest_cache/
 **__pycache__/
+/db.sqlite3
```

Comment:
Ad blank line to the end of the files in the future

Reply:
Ok